### ISSUE TITLE
[BUG] The website is nor responsive 

### ISSUE THREAD

**Issue by AdarshRawat1** [OpenCodeEra/OpenCodeEra#14]
**Describe the bug**
The website is not yet responsive for smaller screens (mobile phones). 

**Expected behavior**
Responsive website

**Comment by CodeTorso** [comment]
Give me 24 hours time. Working on it.

**Comment by AdarshRawat1** [comment]
@CodeTorso you can divide this issue into smaller issues so that you can get some assistance from others as well.

**Comment by CodeTorso** [comment]
Forgot that issue.

Give me 24 more hours. Will be making exactly 4 PRs you know why ?? 😎😎

**Comment by AdarshRawat1** [comment]
> Forgot that issue.

If you say so 😁
> Give me 24 more hours. Will be making exactly 4 PRs you know why ?? 😎😎

Waiting 🚀

**Comment by CodeTorso** [comment]
Didn't you  say we need a new start

Shall I build a website from scratch ?

**Comment by CodeTorso** [comment]
and already this is hacktoberFest so many new contributors will join this

**Comment by CodeTorso** [comment]
I can share this website on their Discord if you want

**Comment by AdarshRawat1** [comment]
> and already this is hacktoberFest so many new contributors will join this

Great initiative! !! highly appreciated.

**Comment by CodeTorso** [comment]
I promise it will have the basic design by that time if you like the idea

**Comment by CodeTorso** [comment]
So should I start from scratch deleting the website files but not github files

**Comment by AdarshRawat1** [comment]
> Shall I build a website from scratch?

Yes, do let me know if you need any assistance. Make sure to include a Hacktoberfest tag and some quality good first issues as well.
 
p.s. Answering your query regarding maintainers on the new project - 
You'll be assigned as a maintainer as you initiated the project.

**Comment by CodeTorso** [comment]
This issue has been waiting for too long now. It's time for it to get finished.

Making some PRs today that I hope will do that job.

Considering to use tailwind CSS

**Comment by AdarshRawat1** [comment]
> Making some PRs today that I hope will do that job.

 Kindly take a look at #73 to avoid conflicts.
> Considering to use tailwind CSS

It might pose a challenge for a few individuals to contribute.

**Comment by AdarshRawat1** [comment]
Hey @CodeTorso, @Kamalkoranga, @Vijay-Gunwant, and @Ritik-Bhasarkar! How about we break down this task into smaller issues? This way, everyone can contribute more effectively.

**Comment by Kamalkoranga** [comment]
Yeah.... Which part should I do ?

**Comment by AdarshRawat1** [comment]
@Kamalkoranga Simply choose a section, open an issue, and assign it to yourself. 

- you can also open small `good first issues` for newcomers. (Keep them simple to understand by defining solutions along with issue)

**Comment by Vijay-Gunwant** [comment]
> Hey @CodeTorso, @Kamalkoranga, @Vijay-Gunwant, and @Ritik-Bhasarkar! How about we break down this task into smaller issues? This way, everyone can contribute more effectively.

Sure please tell me if can be of any help

**Comment by jaydippatel83** [comment]
I want to make the home page responsive. Could you please assign this task to me?
